packages/service-library/src/servicelib/fastapi/prometheus_instrumentation.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=protected-access
+
+
+from fastapi import FastAPI
+from prometheus_fastapi_instrumentator import Instrumentator
+
+
+def instrument_app(app: FastAPI):
+
+    instrumentator = (
+        Instrumentator(
+            should_instrument_requests_inprogress=True, inprogress_labels=False
+        )
+        .instrument(app)
+        .expose(app, include_in_schema=False)
+    )
+
+    def _unregister():
+        for collector in list(instrumentator.registry._collector_to_names.keys()):
```

Comment:
You do not need `list` here. actually you can directly iterate as
```python
for collector in instrumentator.registry._collector_to_names:
   ...
```

Reply:
done